plugins/europeana/entity.js

```diff
@@ -182,3 +182,39 @@ function getRelatedEntityTitleLink(entities) {
   }
   return entityDetails;
 }
+
+/** Get the description for the entity
```

Comment:
Put the description of the function on the following line, i.e. `/**` should be on its own line.